### ISSUE TITLE
All accessories no response

### ISSUE THREAD

**Issue by arjena** [homebridge/homebridge#3378]
### Current Situation

Without changing anything, my home app does not connect to homebridge anymore. I did several things to resolve this, but to no avail. 
I deleted the home in the home app, made a new home, added homebridge. Everything works. An hour later, after I had been away and therefore disconnected from my network for a while, no response. Rebooting the router did nothing. Important to mention is that I have no hub in this home. In my other home there is an Apple TV set up as a hub and there everything works as expected. Apart from the hub, the setup in both homes is identical.
The home app (both on my iPhone and computer) complains that I need a hub for to have access while away from home. As the RPi running homebridge is wire-connected to the router and my phone to the same router via WiFi, I am not away from home in any way. 
When I log in to the homebridge web page everything is OK and I can interact with my accessories just fine.
This setup worked fine for over two years without a hub. Am I missing something or has Apple once again found a new way to get me to buy new Apple stuff?

### Logs

```markdown
Not important
```

### Configuration

```json
Not important
```

### Environment

- OS: 
- Software: Homekit on iOs and OSX
- Node: 
- npm:

### Process Supervisor

systemd

### Additional Context

No hub

**Comment by ZZPeter** [comment]
I have a similar issue and it happens out of nowhere. Just after starting, all accessories can update once in the Home App. But then all accessories cannot connect to home bridge anymore. All bridges based on home bridge do not react anymore.

**Comment by skumancer** [comment]
I'm having the exact same issue. My accessories stop responding every couple of hours, so I have to log into homebridge and they start working again. I've changed nothing that would make this happen, and I've even reverted those changes to see if they help, to no avail. I've also soft-reset all components along the way (Homepod, Routers, iPhone, etc).

The only change done on my side is upgrade to iOS 17 Beta 2/3, but this is happening across the whole home, not just on iOS 17 devices.

**Comment by arjena** [comment]
First of all, thank you ZZPeter for pointing me in the right direction.
Took a while, but I found the source of my problems. After going through the logs of my router, I found that a device on my network was resetting IGMP from V3 to V2. Then, after a couple of minutes, the router reset it to V3 again, only to be set back to V2 after a coupe more minutes. Since the device in question did not display a name but only an IP number it took a lot of trial and error to find it. And just when I thought I had checked every device in the house I realised I had a WiFi extender in the garden. Turning that off did the trick. Now Home is stable again. And I don't have WiFi in the garden anymore, but that's another problem ;). 
Still don't get when and how this started though. The WiFi extender has been there long before I installed Homebridge in the house and never gave me problems. Most likely the router is to blame. It is set to auto update its firmware (required by my ISP) so maybe the switching from IGMP did not happen before. But of course that's just guessing, I'll never know for sure.
